doc/conf.py

```diff
@@ -68,6 +68,7 @@
 autosummary_generate = True
 autodoc_default_flags = ['members', 'inherited-members']
 add_module_names = False
+numpydoc_class_members_toctree = False
```

Comment:
see https://stackoverflow.com/a/66139873

This feels a bit like a nuclear option but have not found a better way to deal with those warnings